hoomd/hpmc/tune/move_size.py

```diff
@@ -259,6 +238,30 @@ def _update_tunables_attr(self, attr, value):
         for tunable in self._tunables:
             setattr(tunable, attr, value)
 
+    def _target_postprocess(self, target):
+        if not (0 <= target <= 1):
+            return target
```

Comment:
Is this `return` a mistake?
```suggestion
```

Reply:
Yes, I do not know how that happened, maybe a botched copy and paste. Good catch.